hawk/app/views/layouts/application.html.haml

```diff
@@ -7,17 +7,17 @@
     .navbar.navbar-fixed-top.navbar-inverse{ role: "navigation" }
       .container-fluid{ style: "white-space: nobreak;" }
         .navbar-header
-          = link_to image_tag("head/white.png", size: "160x23"), root_path, class: "navbar-brand"
+          = link_to image_tag("head/white.png", size: "120x40"), root_path, class: "navbar-brand"
```

Comment:
I think we should get entirely rid of these dimensions and provide the images in the correct size.